sentry-types/src/protocol/v7.rs

```diff
@@ -1415,6 +1421,12 @@ impl fmt::Display for TraceId {
     }
 }
 
+impl fmt::Debug for TraceId {
+    fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
+        write!(fmt, "{}", self)
```

Comment:
Likewise here, add the type to the debug output:


```suggestion
        write!(fmt, "TraceId({})", self)
```

This likely also belongs in a separate PR with the SpanId change